typecats/tc.py

```diff
@@ -18,7 +18,8 @@
     _emit_exception_to_default_handler,
     TypecatsCommonExceptionHook,
 )
-from .strip_defaults import get_stripping_converter
+from .strip_defaults import ShouldStripDefaults
```

Comment:
Why does this look like a class?

Reply:
This seems to be the generally agreed upon but undocumented(?) convention for global context variables. They are not constants, so full uppercase does not apply. Having them CamelCase makes them stand out more than regular snake_case <things>, as it should considering that most of them usually end up changing code paths (having other effects) in underlying functions when set. I wish I had a proper explanation, but I don't. 😕 To me, this just feels more natural for some reason in a way that's hard to put into words?

Peter used this convention in xoto3's contextvars' unit tests (and wrote this change in this PR too) so at least our usage is consistent throughout
https://github.com/xoeye/typecats/pull/13/files/ef712a3315b217a4107b230eabaff08d6304b2bc#diff-8baebb99d0e61ba52c01ec93871d426fac65e5a1fb735a1891d305b19e3811deR43